test/LANDRegistry.js

```diff
@@ -513,38 +514,46 @@ contract('LANDRegistry', accounts => {
         yNewUser.length.should.be.equal(0)
       })
 
-      it('does not transfer lands if it is called by not authorized operator', async function() {
+      it('does not transfer LANDs if it is called by not authorized operator', async function() {
         const [xUser, yUser] = await land.landOf(user)
         await assertRevert(
-          land.transferManyLand(xUser, yUser, creator, { from: operator })
+          land.transferManyLand(xUser, yUser, creator, sentByOperator)
         )
       })
 
-      it('does not transfer lands if land does not exist', async function() {
+      it('does not transfer LANDs if land does not exist', async function() {
         await assertRevert(
           land.transferManyLand([12, 4], [1, 2], creator, sentByUser)
         )
       })
 
-      it('does not transfer lands if x length is not equal to y length', async function() {
+      it('does not transfer LANDs if x length is not equal to y length', async function() {
         await assertRevert(
           land.transferManyLand([0, 0], [0, 1, 3], creator, sentByUser)
         )
       })
     })
   })
 
-  describe('transfer land to estate', function() {
+  describe('transfer LAND to estate', function() {
     let estateId
 
     beforeEach(async function() {
       await land.assignMultipleParcels([3], [3], creator, sentByCreator)
-      estateId = await createEstate([3], [3], creator, sentByCreator)
+      estateId = await createEstate([3], [3], user, sentByCreator)
     })
 
     describe('transferLandToEstate', function() {
-      it('transfers land to an Estate if it is called by owner', async function() {
+      it('doesnt transfer the LAND to an Estate if it is owned by the sender', async function() {
```

Comment:
I would rewrite as "do not transfer LAND to an Estate if is not owned by the sender"

Reply:
but wouldn't that read as:
`it do not transfer LAND ...` ?

or, in the mocha logs:
```
transferLandToEstate
  do not transfer LAND
```

which I think it's not grammatically correct. Maybe we can change it for `it('should not transfer ...')`

Reply:
👍